tardis/montecarlo/base.py

```diff
@@ -220,3 +222,44 @@ def calculate_f_nu(self, frequency):
 
     def calculate_f_lambda(self, wavelength):
         pass
+
+    def property_to_hdf(self, property):
```

Comment:
This needs some kind of change anyway. It does the same thing as `BasePlasmaProperty._get_output_to_hdf`. I'll probably introduce a similar method to `util` and use them everywhere needed.